Sources/PeckerKit/Configuration.swift

```diff
@@ -87,10 +97,28 @@ public struct Configuration {
                   excludedGroupName: yamlConfiguration?.excludedGroupName ?? [],
                   blacklistFiles: yamlConfiguration?.blacklistFiles ?? [],
                   blacklistSymbols: yamlConfiguration?.blacklistSymbols ?? [],
-                  outputFile: outputFilePath)
+                  outputFile: outputFilePath,
+                  xcodeAppPath: absoluteXcodeAppPath)
     }
 }
 
+private func createXcodeAppPath(pathFromOptions: String?, pathFromConfig: String?, fileSystem: FileSystem) -> AbsolutePath {
+    if let absolutePath = absolutePathIfExists(path: pathFromOptions, fileSystem: localFileSystem) {
+        return absolutePath
+    } else if let absolutePath = absolutePathIfExists(path: pathFromConfig, fileSystem: localFileSystem) {
+        return absolutePath
+    } else {
+        return AbsolutePath("/Applications/Xcode.app")
```

Comment:
Now that we have a command line options, we could move the default value into `Sources/Pecker/PeckerCommand.swift`